src/SDK/Client/Client.ts

```diff
@@ -95,6 +95,7 @@ export class Client {
 
             this.wallet = new Wallet({
```

Comment:
We need to check that if `this.options.wallet.network` is defined it should be equal to `this.network`.